tests/unit/sstring_test.cc

```diff
@@ -238,3 +238,14 @@ BOOST_AUTO_TEST_CASE(test_resize_and_overwrite) {
         BOOST_CHECK_EQUAL(s, sstring(smaller_size, pattern));
     }
 }
+
+
+#ifdef __cpp_lib_three_way_comparison 
+
+BOOST_AUTO_TEST_CASE(test_compares_left_hand_not_string) {
+    // mostly a compile test for non-sstring left-hand-side
+    BOOST_REQUIRE("a" < sstring("b"));
```

Comment:
nit, use
```c++
BOOST_REQUIRE_LT("a", sstring("b"));
```

Reply:
It was intentional. I'm specifically testing "<" here, so I don't want to delegate what I'm testing to boost.

If I'm testing some computation, then BOOST_REQUIRE_LT(some_computation(), 17) is fine, since it let's boost report the actual value of the computation. I'm not interested in the value of "a" or sstring("b").